IdentityCore/src/auth_scheme/MSIDAuthenticationSchemePop.m

```diff
@@ -64,14 +66,23 @@ - (instancetype)initWithSchemeParameters:(NSDictionary *)schemeParameters
         }
         
         NSString *kidJwk = [_req_cnf msidBase64UrlDecode];
+        if (!kidJwk)
```

Comment:
nit: can we use msidIsStringNilOrBlank:kidJwk here as well for consistency.

Reply:
Addressed. Updated validation checks in MSIDAuthenticationSchemePop using [NSString msidIsStringNilOrBlank:] for consistency